src/dhcp_fsm.erl

```diff
@@ -335,12 +335,12 @@ delegate(F, Pkg, Opts, State = #state{handler = M}) ->
                     Dst = reply_addr(R),
                     {ok, Bin} = dhcp_package:encode(R),
                     gen_udp:send(State#state.socket, Dst, 68, Bin),
-                    {ok, R, State#state{handler_state = S1, last=erlang:now()}};
+                    {ok, R, State#state{handler_state = S1, last=current_time()}};
                 false ->
                     lager:error("[DHCP] invalid reply package for ~p:~p -> ~p", [M, F, R])
```

Comment:
Line 340 is too long:                     lager:error("[DHCP] invalid reply package for ~p:~p -> ~p", [M, F, R]).